### ISSUE TITLE
cassandra plugin - batch requests throws error

### ISSUE THREAD

**Issue by cohesivejones** [DataDog/dd-trace-js#494]
I just tried to add dd-trace@v0.10.1 to my project and experienced the following error while calling a batch on the cassandra client:

`callback.apply is not a function()`

```
// dd-trace/src/plugins/cassandra-driver.js

function createWrapBatch (tracer, config) {
  return function wrapBatch (batch) {
    return function batchWithTrace (queries, options, callback) {
      const query = combine(queries)
      const span = start(tracer, config, this, query)
      const scope = tracer.scope()

      callback = scope.bind(callback || options)

      return scope.bind(batch, span).call(this, queries, options, function (err) {
        finish(span, err)
        return callback && callback.apply(this, arguments)  //<-- throws error callback.apply is not a function()
      })
    }
  }
}
```

**Dependencies:**
cassandra-driver: 3.5.0
dd-trace: 0.10.1

Currently we are using dd-trace 0.9.0 without issue

**Comment by rochdev** [comment]
We added the `cassandra-driver` integration in 0.10.0. While this is being fixed, you can disable it with `tracer.use('cassandra-driver', false)`.

Can you share the line of code where you call `batch`?

**Comment by cohesivejones** [comment]
```
Cassandra.client.batch(
    [{
      "query":"update pinnedHeadlines set details = {{headlineid: 6352e7b0-1a34-11e9-843c-6371745f9429, pinned: 10}}, version = toTimestamp(now()) where id = 2",
      "params":[]
    }, {
      "query":"update headlineDetails set title = ?, tenant = ?, translations = ?, headlineOwner = ?, contentOwner = ?, status = ?, startTime = ?, endTime = ?, limitsTo = ?, tags = ?, images = ?, imageMetadata = ?, notes = ?, createdBy = ?, createdTime = ?, lastUpdatedBy = ?, lastUpdatedTime = ?, priority = ?, priorityExplanation = ?, channels = ? where headlineid = ?",
      "params":["Sample Title", "comms", ...]
    }],
    {"prepare":true}
)
```

**Comment by rochdev** [comment]
Fix is on the way. In the meantime, if you want to keep the plugin enabled and fix the issue, simply pass an empty callback as a third argument to `Cassandra.client.batch`.

**Comment by rochdev** [comment]
Fix released in 0.10.2
